packages/fluentui/react-northstar/src/components/Dropdown/DropdownSelectedItem.tsx

```diff
@@ -132,57 +135,61 @@ const DropdownSelectedItem: React.FC<WithAsProp<DropdownSelectedItemProps>> &
     },
   });
 
-  const contentElement = Box.create(header, {
+  const contentElement = Box.create(
+    Box.create(header, {
+      defaultProps: () => ({
+        as: 'span',
+        className: dropdownSelectedItemSlotClassNames.header,
+        styles: resolvedStyles.header,
+      }),
+    }),
+    {
+      defaultProps: () => ({
+        className: dropdownSelectedItemSlotClassNames.header,
+        styles: resolvedStyles.content,
+      }),
+    },
+  );
+
+  const iconElement = Box.create(icon, {
     defaultProps: () => ({
-      as: 'span',
-      className: dropdownSelectedItemSlotClassNames.header,
-      styles: resolvedStyles.header,
+      'aria-label': `Remove ${header} from selection.`, // TODO: Extract this in a behaviour.
+      className: dropdownSelectedItemSlotClassNames.icon,
+      styles: resolvedStyles.icon,
     }),
+    overrideProps: handleIconOverrides,
   });
 
-  const iconProps = _.isNil(icon)
-    ? icon
-    : {
-        name: null,
-        children: (ComponentType, props) =>
-          Box.create(icon, {
-            defaultProps: () => ({
-              'aria-label': `Remove ${header} from selection.`, // TODO: Extract this in a behaviour.
-              className: dropdownSelectedItemSlotClassNames.icon,
-              styles: resolvedStyles.icon,
-            }),
-            overrideProps: handleIconOverrides(props),
-          }),
-      };
-
-  const imageProps = _.isNil(image)
-    ? image
-    : {
-        children: (ComponentType, props) =>
-          Image.create(image, {
-            defaultProps: () => ({
-              avatar: true,
-              className: dropdownSelectedItemSlotClassNames.image,
-              styles: resolvedStyles.image,
-            }),
-            overrideProps: props,
-          }),
-      };
+  const imageElement = Box.create(
+    Image.create(image, {
+      defaultProps: () => ({
+        avatar: true,
+        className: dropdownSelectedItemSlotClassNames.image,
+        styles: resolvedStyles.image,
+      }),
+    }),
+    {
+      defaultProps: () => ({
+        className: dropdownSelectedItemSlotClassNames.image,
+        styles: resolvedStyles.media,
```

Comment:
I don't see styles for this element. Is it required to have it in markup?

Reply:
Removed